csharp/src/sdk/Geometry/Polygon.cs

```diff
@@ -200,10 +210,153 @@ internal static List<IntPoint> ToClipperPath(this Polygon p)
             return path;
         }
 
+        /// <summary>
+        /// Construct a Polygon from a clipper path 
+        /// </summary>
+        /// <param name="p"></param>
+        /// <returns></returns>
+
         internal static Polygon ToPolygon(this List<IntPoint> p)
         {
             var scale = 1024.0;
             return new Polygon(p.Select(v=>new Vector3(v.X/scale, v.Y/scale)));
         }
+
+        /// <summary>
+        /// Returns true if the interior of the first polygon is coincident with every interior and boundary point of the second polygon.
+        /// </summary>
+        /// <param name="p1">The polygon boundary.</param>
+        /// <param name="p2">The polygon tested for containment within the boundary.</param>
+        /// <param name="cvr">If true, polygon is contained if boundary points are coincident.</param>
```

Comment:
Where possible use `Perimeter` in place of `Boundary` and `Vertex` or `Vertices` in place of `Point`. This is the nomenclature used elsewhere in `Polygon`.